TESTING.md

```diff
@@ -90,13 +107,13 @@ KIND_CONTEXT=e2e make test-e2e-images
 
 #### Using podman with Kind
 
-When running kind on MacOS using podman, it is recommended to create podman machine with `4` CPUs and `8 GiB` memory. Less resources might cause end to end tests to fail because of lack of resources in the cluster.
+When running Kind on MacOS using [podman](https://podman.io), it is recommended to create podman machine with `4` CPUs and `8 GiB` memory. Less resources might cause end to end tests to fail because of lack of resources in the cluster. If you are using [podman-desktop](https://podman-desktop.io/docs/podman/creating-a-podman-machine), it is quite easy to create the podman machine or update the machine resource settings.
```

Comment:
```suggestion
When running Kind on MacOS using [podman](https://podman.io), it is recommended to create podman machine with `4` CPUs and `8 GiB` memory. Less resources might cause end to end tests to fail because of lack of resources in the cluster. If you are using [podman-desktop](https://podman-desktop.io/docs/podman/creating-a-podman-machine), you can create (or update) the podman machine like this:
```

Reply:
Podman desktop has UI so instead of commands listed below, can achieve same easily from UI. I thought mentioning that would be good 🙂 wdyt?

Reply:
Ah ok I got confused sorry! Maybe mention podman-desktop after the command?